src/ts/components/subtitleoverlay.ts

```diff
@@ -33,9 +38,24 @@ export class SubtitleOverlay extends Container<ContainerConfig> {
     let subtitleManager = new ActiveSubtitleManager();
     this.subtitleManager = subtitleManager;
 
+    this.configureCea608Captions(player, uimanager);
+
     player.addEventHandler(player.EVENT.ON_CUE_ENTER, (event: SubtitleCueEvent) => {
       let labelToAdd = subtitleManager.cueEnter(event);
-
+      // If there is positionning data we assume we have CEA-608 style captions
+      labelToAdd.isCEA608 = event.position != null;
```

Comment:
Please move all of the CEA-608 related code into `configureCea608Captions`. You can do this by adding a `getCues(event: SubtitleCueEvent): ActiveSubtitleCue[]` method to the `ActiveSubtitleManager`, subscribe to the `ON_CUE_ENTER` event in `configureCea608Captions`, call `getCues` there and apply the necessary CSS properties. Take care that `configureCea608Captions` must be called after subscribing to `ON_CUE_ENTER` in `configure` for the subtitle cue objects to exist when `ON_CUE_ENTER` in `configureCea608Captions` is called.